File: LB/PROgram0045.c
/*

Q45) Write a PROgram which Accept number from user and return count of summation of even and odd digits in number

    input :- 2395
    output :- 270  (2*3*9*5)

*/

#include<stdio.h>

int CheckMulti(int iNo1)
{
    int iCnt = 0;
    int iDigit = 0;
    int iEven = 0;
    int iOdd = 0;

    for(; iNo1 != 0;)
    {
        iDigit = iNo1 % 10;

        if(iDigit % 2 == 0)
        {
            iCnt++;
            
            iEven = iEven + iDigit;
        }

        else
        {
            iCnt++;

            iOdd = iOdd + iDigit;
        }

        iNo1 = iNo1 / 10;
    }

    return iEven - iOdd;
}

int main()
{
    int iValue1 = 0;
    int iRet = 0;

    printf("Enter the Number : ");
    scanf("%d",&iValue1);

    iRet = CheckMulti(iValue1);

    printf("Multiplication of given number digits is : %d",iRet);

    return 0;
}

File: PM/PROgram0045.py
'''

Q5) Write a PROgram that Accept a Number from user And Check whether is it divisible by 3 And 5 or not1

'''

def ChkDivisible(iNo1):
    if(iNo1 % 3 == 0 and iNo1 % 5 == 0):
        print("The Number is Divisible By 3 & 5")
    else:
        print("The Number is <<<Not>>> Divisible By 3 & 5")

def main():
    print("Enter the Number : ")
    iValue1 = int(input())

    ChkDivisible(iValue1)   

if __name__ == "__main__":
    main()        